README.md

```diff
@@ -65,6 +75,13 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 Your Answer:
 ```js
 // write code here
+// Note: I'm not sure that I understand what this question is asking.
+// var doSomething = function(thingToDo){
+//   
+// }
+// var anotherFunction = function(){
+//   doSomething(thingToDo);
+// }
 ```
```

Comment:
This was looking for a function which takes a callback and then in the function body invokes the callback.